rllib/connectors/connector.py

```diff
@@ -42,29 +42,50 @@ def __init__(
         action_space: gym.Space = None,
         view_requirements: Dict[str, ViewRequirement] = None,
         is_policy_recurrent: bool = False,
+        is_atari: bool = False,
+        preprocessing_enabled: bool = True,
     ):
         """Construct a ConnectorContext instance.
 
         Args:
+            config: Algorithm config dict.
             initial_states: States that are used for constructing
                 the initial input dict for RNN models. [] if a model is not recurrent.
+            observation_space: a policy's observation space.
             action_space_struct: a policy's action space, in python
                 data format. E.g., python dict instead of DictSpace, python tuple
                 instead of TupleSpace.
+            view_requirements: a policy's view requirements.
+            is_policy_recurrent: whether the policy is recurrent.
+            is_atari: whether the policy is for an Atari env.
+            preprocessing_enabled: whether automatic preprocessing is enabled.
         """
         self.config = config or {}
         self.initial_states = initial_states or []
         self.observation_space = observation_space
         self.action_space = action_space
         self.view_requirements = view_requirements
         self.is_policy_recurrent = is_policy_recurrent
+        self.is_atari = is_atari
+        self.preprocessing_enabled = preprocessing_enabled
 
     @staticmethod
-    def from_policy(policy: "Policy") -> "ConnectorContext":
+    def from_policy(
+        policy: "Policy",
+        *,
+        is_atari: bool = False,
+        preprocessing_enabled: bool = True,
+    ) -> "ConnectorContext":
         """Build ConnectorContext from a given policy.
 
         Args:
             policy: Policy
+            is_atari: Whether the policy is for an Atari env.
```

Comment:
I think this is sketchy - same goes for the code we have in RW for auto-wrapping atari envs.
Whether an env is atari or not should not leak into these components.

Reply:
would you feel better if we pass clip_reward instead?

Reply:
I think it would be best to merge this and backlog that we need a cleanup PR for the atari-related stuff.

Reply:
from my perspective, having these additional fields isn't THAT bad (not ideal for sure).
like you have config, and you have context. Context provides required data bits that are not part of config, e.g. view requirements etc.
the part bothering me more is that I feel like these knowledge should be available from policy itself ...

Reply:
yeah it should have been part of the `policy.config` object.

Reply:
I also don't like the `is_atarti` flag. if we change it to `clip_reward` I only have this concern that it should be part of the `policy.config`.